_posts/2020-02-21-newpipe-0.18.4-released.md

```diff
@@ -0,0 +1,58 @@
+
+## Kodi button
+
+If you read this heading and thought "Hey. I have this Kodi button enabled in my settings. Do I get another button?" Damn right you get. Presenting the in-player-kodi-button:
+
+<img class="no-flow" src="/img/screenshots/kodi_player_button.jpg"/>
+
+
+## Improvements
+
+Do you know those ReCaptcha images you have to solve when you surf some websites? Some of our users have seen this inside of NewPipe :( While there is no way to let them magically disappear, NewPipe is now able to deal with them properly, thanks to [@Stypoxs](https://github.com/Stypox) developing and [@juanfra684s](https://github.com/juanfra684) testing. [@karkaminski](https://github.com/karkaminski) improved the main view of the app through removing the legacy dot menu next to the magnifying glass. Also included in his changes is the removing of the history sidebar entry if you don't have history enabled (kinda makes sense). 
+
+[@harshlele](https://github.com/harshlele) strikes again, this time improving the way of asking for the permission to display over other apps. Should work properly now on devices running Android 6+. [@B0pol](https://github.com/B0pol) made quite some fixes in his second contribution: PeerTubes videos get full descriptions now as well as markdown applied. PeerTube itself received proper handling of age limits, the recently added feed got fixed (the upload date was off by as many hours as you are from GMT) and its thumbnail is now in high quality. MediaCCCs videos gets valid newlines. And timestamps in YouTube video descriptions work.
+
+Did you ever long press on your local playlists and were bothered by the fact that you are only able to delete, not rename then? Well not anymore, now you can do both in the same menu, thanks to [@XiangRongLin](https://github.com/XiangRongLin). If you notice some slight changes in the english strings in this release, you are correct, [@comradekingu](https://github.com/comradekingu) updated them.
+
+
+
+## Bug fixes
+
+Changing the brightness for the first time while playing a video now doesn't reset itself, [@vhsw](https://github.com/vhsw) did that. [@raphj](https://github.com/raphj) fixed a bug where the video continued to be played when the screen is turned of. [@kapodamy](https://github.com/kapodamy) apparently enjoys working subtitle files as much as the next guy, so there you have them, no more cutting off at line breaks. He didn't stop there, also downloading to SD cards works now on not CTF compliant devices.
+
+Using a KitKat android device will not stop you anymore from downloading subtitles, thanks to [@mqus](https://github.com/mqus).  The third contribution of [@kapodamy](https://github.com/kapodamy) stops the app from treating a corrupt <code>.mp4</code> file as audio. And finally, [@chr56](https://github.com/chr56) made up-to-date Chinese translation possible.
+
+
+### Conclusion
```

Comment:
That's a nice section! Please also mention the hard working translators which needed to take action after comradekingu improved the English strings.

Reply:
thank you. english was higher up though, in improvement. chr56 only changed langcodes and some strings on chinese part, thats why its a fix

Reply:
Or do you mean the conclusion section later on?

Reply:
I just assumed the latter now

Reply:
Yes, I meant the conclusion.

Reply:
Btw, the conclusion heading is smaller then the other headings. It is preceded by three `#` and all other headings by two `#`.

Reply:
I just wanted it to stand out a bit initially, less of a heading, since it wasnt a full paragraph. Then things lead to another and it got bigger, so yeah, I could make it a normal one now

Reply:
fixed